it/common/src/main/java/org/apache/beam/it/common/ResourceManager.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.it.common;
+
+/** Common interface across resource managers. */
+public interface ResourceManager {
```

Comment:
This seems close to `Closeable` or `Autocloseable`. Would it make sense to use it in that style?

Reply:
Changed the resource managers to match `Closable` and `AutoCloseable` style in https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/681. However, some resource managers require an intermediate interface (e.g JDBC) so I have left them as is.